MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -95,6 +97,23 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
         mesh = readBinary(file_name_base);
     }
 
+    auto* vtkGhostType =
+        mesh->getProperties().createNewPropertyVector<std::int8_t>(
+            "vtkGhostType", MeshLib::MeshItemType::Cell, 1);
+    if (!vtkGhostType)
+    {
+        WARN(
+            "Could not create PropertyVector for the cell values of "
+            "vtkGhostType in Mesh.");
+    }
+    else
+    {
+        std::fill_n(std::back_inserter(*vtkGhostType),
```

Comment:
Minor. You could `reserve(_mesh_info.regular_elements + _mesh_info.ghost_elements)` before filling the vector to superb performance ;)

Reply:
Added.